### DISCUSSION TITLE
如果将已有小程序项目渐进的迁移到 Taro 上

### DISCUSSION THREAD

**Discussion by anhulife** [NervJS/taro#10290] [Q & A]
我们团队目前维护了一个包含 53个页面的小程序，这个小程序使用 [minapp](https://qiu8310.github.io/minapp/) 框架开发，现在考虑迁移到 Taro 上，但由于页面较多，而且业务需求不能停，所以直接全量迁移的难度很大。

想问各位大佬，有没有渐进式迁移的方案，可以将页面逐步的迁移到 Taro 上。

**Comment by Chen-jj** [comment]
建议以混合的形式，从分包开始咯：https://docs.taro.zone/docs/next/taro-in-miniapp#%E6%8A%8A-taro-%E9%A1%B9%E7%9B%AE%E4%BD%9C%E4%B8%BA%E4%B8%80%E4%B8%AA%E5%AE%8C%E6%95%B4%E5%88%86%E5%8C%85

**Reply by anhulife** [reply]
👍 ，我先试试

**Reply by airqj** [reply]
你好 测试有结果了吗，mina 与 taro 混合开发是否可行？

**Reply by anhulife** [reply]
嗯，可以混合开发。没有深度体验，暂时没发现什么坑。

**Reply by airqj** [reply]
非常感谢

**Comment by cmdparkour** [comment]
先迁移一个页面过去，然后用低端机测一测原来的性能和迁移后的性能，对比一下，再确定是否迁移吧
